openedx/adg/lms/applications/tests/test_views.py

```diff
@@ -40,6 +37,20 @@ def user_fixture():
     return user
 
 
+@pytest.mark.django_db
+@pytest.fixture(name='user_with_no_hub')
+def user_with_no_hub_fixture():
+    """
+    Create a test user without corresponding ApplicationHub object
```

Comment:
*Create a test user with extended profile and without corresponding ApplicationHub object